pyomo/solvers/tests/mip/test_scip_solve_from_instance.baseline

```diff
@@ -1,7 +1,7 @@
 {
     "Problem": [
         {
-            "Lower bound": -Infinity,
+            "Lower bound": 1.0,
```

Comment:
How is this related? If not, no biggie, just curious.

Reply:
It's not - apart from as I was removing use of `ProblemSense` I ran across this bug in the SCIP interface where we always reported the incumbent objective value as the upper bound (regardless of the optimization sense).  I fixed that (instead of preserving the bug and fixing it in a separate PR) - and then promptly realized that we never returned the opposing (dual) bound, and that we should.  So I fixed that, too.  That latter fix caused the change in the baseline.